js/src/data/actions.js

```diff
@@ -740,13 +760,34 @@ export function* saveTargetAudience( targetAudience ) {
 	}
 }
 
+/**
+ * Adaptes the campaign entity received from API.
+ *
+ * @param {Object} campaign The campaign entity to be adapted.
+ * @return {Campaign} Campaign data.
+ */
+function adapteAdsCampaign( campaign ) {
+	const allowMultiple = campaign.targeted_locations.length > 0;
+	const displayCountries = allowMultiple
+		? campaign.targeted_locations
+		: [ campaign.country ];
+	return {
+		...campaign,
+		allowMultiple,
+		displayCountries,
+	};
+}
```

Comment:
💅  Until now in the `actions.js` file we only have the action functions. Now we also have this auxiliary function for mapping the Ads campaign. Now it's only one. But wondering if in the function we will start adding more and more aux functions together with the actions. 

I'd suggest to have in in an external file

Reply:
Good idea 💯 . Moved in https://github.com/woocommerce/google-listings-and-ads/pull/1273/commits/593dc2e0ab05588aef0b4821ff4c649d8715ef36 with fixing a typo. 😅